src/CommunityVoices/App/Website/Config/Routes.json

```diff
@@ -569,11 +569,28 @@
         }
     },
     "getAllErrors": {
-        "notation": "/errors",
+        "notation": "/errors-log",
         "method": "get",
         "defaults": {
             "resource": "DisplayError",
             "action": "getAllErrors"
         }
+    },
+    "getSomeErrors": {
+        "notation": "/errors-log/{lines}",
+        "method": "get",
+        "defaults": {
+            "resource": "DisplayError",
+            "action": "getSomeErrors"
+        }
+    },
+    "postErrorsSearch": {
+        "notation": "/errors-log/search",
+        "method": "post",
```

Comment:
POST is usually reserved for making changes to the database.  GET is for receiving data.  It does not quite line up here.

Reply:
Meant to get rid of this.